lib/Doctrine/DBAL/Schema/Visitor/DropSchemaSqlCollector.php

```diff
@@ -78,15 +78,18 @@ public function getQueries()
     {
         $sql = [];
 
+        /** @var ForeignKeyConstraint $fkConstraint */
         foreach ($this->constraints as $fkConstraint) {
             $localTable = $this->constraints[$fkConstraint];
             $sql[]      = $this->platform->getDropForeignKeySQL($fkConstraint, $localTable);
         }
 
+        /** @var Sequence $sequence */
         foreach ($this->sequences as $sequence) {
             $sql[] = $this->platform->getDropSequenceSQL($sequence);
         }
 
+        /** @var Table $table */
```

Comment:
Would it make sense to document these on the iterators above?

Reply:
Not as of now. See the description.